inc/packages/namespace.php

```diff
@@ -83,7 +84,7 @@ function get_did_hash( string $id ) {
  * @return DIDDocument|WP_Error
  */
 function get_did_document( string $id ) {
-	$cached = get_site_transient( $id );
+	$cached = wp_cache_get( $id );
```

Comment:
```suggestion
	$cached = wp_cache_get( $id, 'did' );
```
Also, should this be `did-docs` for consistency with `metadata-docs`?

Reply:
I guess it does if it's declared.

Reply:
Or should we just remove it the group?

Reply:
Since this just uses `$id` as the cache key, it's probably best to keep the groups.